index.yaml

```diff
@@ -7313,20 +7313,45 @@ entries:
       catalog.cattle.io/release-name: prometheus-federator
     apiVersion: v2
     appVersion: 0.3.4
-    created: "2024-06-10T06:15:35.190850129-04:00"
+    created: "2024-07-06T07:22:11.856642625-04:00"
     dependencies:
     - condition: helmProjectOperator.enabled
       name: helmProjectOperator
       repository: file://./charts/helmProjectOperator
       version: '*'
     description: Prometheus Federator
-    digest: f3960b8de53ce8344ecff1ee726a4c42579724c1edddb456b08e7db5eb073904
+    digest: bc09acfacb1feb46054c10b6b2cf2262684e5b709757e71cf37766b6e4fcd026
     icon: https://raw.githubusercontent.com/rancher/prometheus-federator/main/assets/logos/prometheus-federator.svg
     kubeVersion: '>=1.26.0-0'
     name: prometheus-federator
     urls:
-    - assets/prometheus-federator/prometheus-federator-104.0.0-rc1+up0.4.0.tgz
-    version: 104.0.0-rc1+up0.4.0
+    - assets/prometheus-federator/prometheus-federator-104.0.0+up0.4.0.tgz
+    version: 104.0.0+up0.4.0
+  - annotations:
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/display-name: Prometheus Federator
+      catalog.cattle.io/kube-version: '>= 1.26.0-0 < 1.31.0-0'
+      catalog.cattle.io/namespace: cattle-monitoring-system
+      catalog.cattle.io/os: linux,windows
+      catalog.cattle.io/permits-os: linux,windows
+      catalog.cattle.io/provides-gvr: helm.cattle.io.projecthelmchart/v1alpha1
+      catalog.cattle.io/rancher-version: '>= 2.9.0-0 < 2.10.0-0'
+      catalog.cattle.io/release-name: prometheus-federator
+    apiVersion: v2
+    appVersion: 0.3.5
+    created: "2024-07-06T07:24:55.447885286-04:00"
+    dependencies:
+    - condition: helmProjectOperator.enabled
+      name: helmProjectOperator
+      repository: file://./charts/helmProjectOperator
+      version: 0.2.1
+    description: Prometheus Federator
+    digest: 6692b3ab72f05aa7564c84513157386772c06b5c06d865779785a9766e9ecb5d
+    icon: https://raw.githubusercontent.com/rancher/prometheus-federator/main/assets/logos/prometheus-federator.svg
+    name: prometheus-federator
+    urls:
+    - assets/prometheus-federator/prometheus-federator-104.0.0-rc1+up0.4.2.tgz
+    version: 104.0.0-rc1+up0.4.2
```

Comment:
Added a second time here. Already exists on Line 7303 - can't comment on it though, I guess since its just above (outside) the change set?